### PR TITLE
forsaken skills

### PR DESCRIPTION
**Pull request by Gamemechanicwow**
## 🍰 Pullrequest

Allows spell [7054 forsaken-skills](https://www.wowhead.com/classic/spell=7054/forsaken-skills) to proc [7053 forsaken-skill-shadow](https://www.wowhead.com/classic/spell=7053/forsaken-skill-shadow)

### Proof

- From the comments following the code, it looks like 7053 was mistaken for 7052
- I have verified that all 15 spells exist in the clients from 1.2.4 to 1.12.1 - and spell 7052 exists in none of them

### Issues

A different approach of fixing this, could be by using PickRandomValue(). 

### How2Test

- None

### Todo / Checklist

- [X] None

### CODE REVIEW

**Comment by ratkosrb** [comment]
Why is there a log statement in this function?

**Comment by Gamemechanicwow** [comment]
For no apparent reason. Let's get rid of it.

**Comment by ratkosrb** [comment]
Same spell is used in Aura::TriggerSpell btw.

**Comment by ratkosrb** [comment]
Can you move the variable declaration to the proper scope btw, no idea why its declared outside the switch case.

File: src/game/Spells/SpellAuras.cpp [hunk 1]
```diff
@@ -1457,6 +1457,9 @@ void Aura::TriggerSpell()
         {
             case 7054:
                 spellRandom = urand(0, 14) + 7038;
+                if (spellRandom == 7052)
+                    spellRandom = 7053;
+
                 target->CastSpell(target, spellRandom, true, nullptr, this);
                 return;
                 break;
```
Comment (ratkosrb) [inline src/game/Spells/SpellAuras.cpp]: no need for break after return

Comment (Gamemechanicwow) [reply src/game/Spells/SpellAuras.cpp]: fixed

**Comment by 0blu** [comment]
Maybe adding a comment to those `if`s, so it the future someone still knows why this spellid is not valid and why we need to skip this number.

**Pull request merged**